src/js/engagement_view/src/components/SideBarContent.tsx

```diff
@@ -44,77 +48,127 @@ const useStyles = makeStyles({
     lensName: {
         fontSize: "16px",
     },
+    pagination: {
+        margin: ".5rem",
+        backgroundColor: "#595959",
+    }
 });
 
-type SelectLensProps = {
-    lens: string,
-    score: number,
-    uid: number,
-    lens_type: string,
-    setLens: (lens: string) => void,
-}
 
 function SelectLens(props: SelectLensProps) {
     const classes = useStyles();
-
     return (
         <>
-                <TableRow key={props.lens}>
-                        <TableCell component="th" scope="row">
-                        <Button className = {classes.lensName}
-                            onClick={
-                                () => { 
-                                    props.setLens(props.lens)    
-                                }
-                        }>
-                            {/* #TODO: change color of lense name based on score */}
-                            {props.lens_type + " :\t\t" + props.lens + "\t\t" + props.score}
-                        </Button>
-                        </TableCell>
-                    </TableRow>
+            <TableRow key={props.uid}>
+                <TableCell component="th" scope="row">
+                <Button className = {classes.lensName}
+                    onClick={
+                        () => { 
+                            props.setLens(props.lens)    
+                        }
+                }>
+                    {/* #TODO: change color of lens name based on score */}
```

Comment:
Can we get an issue opened to track this?

Reply:
Created issue: https://github.com/grapl-security/grapl/issues/132